doc/modules/tree.rst

```diff
@@ -500,6 +500,37 @@ Entropy:
 
     H(Q_m) = - \sum_k p_{mk} \log(p_{mk})
 
+
+.. note::
+
+  The entropy criterion computes the Shannon entropy of the categorical
+  distribution parametrized by the class frequency the training data points that
+  reached a given leaf `m`. Using the **Shannon entropy as tree node splitting
+  criterion is equivalent to minimizing the cross-entropy loss** (also known as
+  the categorical deviance or the log-loss) between the true labels :math:`y_i`
+  and the probalistic predictions :math:`T(x_i)` of the tree model :math:`T`.
+
+  To see, this, first recall that the cross-entropy of a tree model :math:`T`
```

Comment:
```suggestion
  To see, this, first recall that the log loss of a tree model :math:`T`
```

In addition, should we calculate the average/mean loss instead of the sum? The final expression would then become very similar to "G".

Reply:
Alright, let me do this. I was about to push the normalized version.